mappings/net/minecraft/entity/projectile/ProjectileEntity.mapping

```diff
@@ -1,13 +1,17 @@
 CLASS net/minecraft/class_1665 net/minecraft/entity/projectile/ProjectileEntity
 	CLASS class_1666 PickupPermission
 		METHOD method_7458 fromOrdinal (I)Lnet/minecraft/class_1665$class_1666;
+			ARG 0 ordinal
 	FIELD field_7571 damage D
 	FIELD field_7572 pickupType Lnet/minecraft/class_1665$class_1666;
 	FIELD field_7573 PROJECTILE_FLAGS Lnet/minecraft/class_2940;
 	FIELD field_7574 shake I
+	FIELD field_7575 punchLevel I
```

Comment:
`punch` is clear enough (can imply an integer field) compared to `PIERCING` (which implies a boolean field)

Reply:
https://github.com/FabricMC/yarn/pull/880/commits/1aa7f82b723458a80bb0f7a6bbe2198bfffefa22#diff-c6066eb450a17bdb67b5f9b78f7c805eR9

Reply:
I changed according to your suggestion, but the code still uses PierceLevel